tests/test-coreos.sh

```diff
@@ -10,6 +10,14 @@ IMAGE_PATH=/tmp/rhcos-latest-image
 SCRIPT_FOLDER=$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P)
 RHCOS_SLB_REPO_PATH=${SCRIPT_FOLDER%/*}
 
+set_test_timeout() {
+	timeout_in_minutes=$1
+
+	for test_name in ${TESTS_LIST}; do
+		sed -i "s|\(Name:.*\"${test_name}\",\)|\1\n\t\tTimeout:     ${timeout_in_minutes} * time.Minute,|g" ${TMP_COREOS_ASSEMBLER_PATH}/${RHCOS_SLB_TEST_PATH}
```

Comment:
Could we force sed to fail in case there is no replace?

https://stackoverflow.com/questions/15965073/return-value-of-sed-for-no-match/15966279

So if somebody changes the test and this sed stops doing what it should, we will know.

Reply:
great idea